### PR TITLE
fix(omid): location and permissions of the tso logging directory 

### PR DESCRIPTION
**Pull request by razvan**
# Description

Successfully tested on OpenShift 4.15:

```
--- PASS: kuttl (295.70s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/omid_hbase-2.4.18,docker.stackable.tech_sandbox_hbase_2.4.18-stackable0.0.0-dev_hdfs-latest-3.3.6_zookeeper-latest-3.9.2_openshift-true_omid-1.1.0,docker.stackable.tech_sandbox_omid_1.1.0-stackable0.0.0-dev (295.03s)
PASS
```

- [x] TODO: set the property `omid.log.dir` to a proper location. **Update** `OMID_OPTS` anv var and set `omid.log.dir` to `/stackable/logs`

The owner of the tso folder is root:
```
stackable@8821649a65ad /stackable $ ls -ltrAh
total 1.2M
drwxr-xr-x. 1 root      root         8 Jul 11 01:37 omid-tso-server-1.1.0
drwxr-xr-x. 1 root      root        60 Jul 11 01:37 omid-examples-1.1.0
...
```

But the tso server runs as `stackable` and tries to write to omid.log in that folder (by default).

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Integration tests ran successfully
```

<details>
<summary>TIP: Running integration tests with a new product image</summary>

The image can be built and uploaded to the kind cluster with the following commands:

```shell
bake --product <product> --image-version <stackable-image-version>
kind load docker-image <image-tagged-with-the-major-version> --name=<name-of-your-test-cluster>
```

See the output of `bake` to retrieve the image tag for `<image-tagged-with-the-major-version>`.
</details>

### CODE REVIEW

File: druid/versions.py [hunk 1]
```diff
@@ -1,25 +1,20 @@
 versions = [
-  {
```
[the remark this answers is not in the document]
Comment (razvan) [reply druid/versions.py]: it's been reverted

File: omid/Dockerfile [hunk 2]
```diff
@@ -8,16 +8,22 @@
 FROM stackable/image/java-devel AS builder
 
 ARG PRODUCT
+ARG DELETE_CACHES="true"
 
 USER stackable
 WORKDIR /stackable
 
-RUN curl --fail -L https://dist.apache.org/repos/dist/release/phoenix/phoenix-omid-${PRODUCT}/phoenix-omid-${PRODUCT}-src.tar.gz | tar -xzC .
+RUN --mount=type=cache,id=maven,uid=1000,target=/stackable/.m2/repository <<EOF
```
[the remark this answers is not in the document]
Comment (razvan) [reply omid/Dockerfile:17]: Done https://github.com/stackabletech/docker-images/pull/773/commits/ba58c27069c4aeda45db3b3f3d9f56b3edffd52d

File: omid/Dockerfile [hunk 3]
```diff
@@ -8,16 +8,22 @@
...
+  curl --fail -L https://dist.apache.org/repos/dist/release/phoenix/phoenix-omid-${PRODUCT}/phoenix-omid-${PRODUCT}-src.tar.gz | tar -xzC .
```
[the remark this answers is not in the document]
Comment (razvan) [reply omid/Dockerfile:18]: thanks

**Review by soenkeliebau — APPROVED** [review]
Looks good to me, logs directory is present and belongs to the stackable user, no logs in there, but I'm not sure if omid ever actually puts something there..

Ran the test in stackabletech/omid-tester and it worked against an image built from this branch.

**Pull request merged**